crates/configuration/src/network.rs

```diff
@@ -393,17 +397,20 @@ mod tests {
             .with_parachain(|parachain1| {
                 parachain1
                     .with_id(1000)
+                    .with_chain("myparachain1")
+                    .with_initial_balance(100_000)
                     .with_collator(|collator| {
                         collator
                             .with_name("collator")
                             .with_command("invalid command")
                             .validator(true)
                     })
-                    .with_initial_balance(100_000)
             })
             .with_parachain(|parachain2| {
                 parachain2
```

Comment:
closure is only inside the `( )` scope so no need to alter the name every time
```suggestion
            .with_parachain(|parachain| {
                parachain
```

Reply:
Yes but I think it's better for readability + only present in tests so that's fine imho

Reply:
I think its unecessery. Preserving the same pattern makes readability easier. not a blocking for the approval - just a proposal

Reply:
Done